common/util.go

```diff
@@ -79,6 +80,11 @@ const (
 	retryKafkaOperationMaxInterval        = 10 * time.Second
 	retryKafkaOperationExpirationInterval = 30 * time.Second
 
+	defaultInitialIntervalInSeconds   = 1
```

Comment:
@mfateev can you confirm if these are the defaults you have in mind?

Reply:
Yes, these look fine.